config/default/kustomization.yaml

```diff
@@ -1,5 +1,5 @@
 # Adds namespace to all resources.
-namespace: model-registry-operator-system
+namespace: opendatahub
```

Comment:
I note the operator will move into the ODH namespace operator

Reply:
Yes, we have to align it with other odh components, which all use the same namespace. 
But, I'll investigate whether we can keep the current kubebuilder default config and use an overlay instead for odh. That way the operator can be used without having to do things the _odh_ way.